### ISSUE TITLE
Problem after installation

### ISSUE THREAD

**Issue by ThibautVerron** [duvholt/i3-pager#16]
Hi,

After following the instructions, I do see a plasmoid I3pager in the list, but the plasmoid does not appear to work: it only shows a kde icon, and on hover it says 

    Error loading QML file: file:///usr/share/plasma/plasmoids/org.kde.I3Pager/contents/ui/main.qml:8:1: module "I3Pager" is not installed

I don't see any error at configuration or compilation, just a few warnings.

Did I miss a step?

Thanks!

**Comment by duvholt** [comment]
If I remember correctly this means that the plasmoid can't find the C++ module. Have you tried restarting plasmashell? I'm not sure if it's necessary but it's worth trying. 
This can be done by running `killall plasmashell; kstart5 plasmashell` or by logging out and in again.

**Comment by ThibautVerron** [comment]
Thanks for the fast reply. I have logged off and on several times, and restarting plasmashell also doesn't help.

I have run the `make -C build install` line again and for some reason it installed (or reinstalled) the libplasmoidplugin.so file, but it still doesn't work (even after removing and readding the plasmoid and restarting plasmashell). 

I have uninstalled, deleted the build directory, rebuilt and reinstalled and now it works.
